src/Microsoft.ML.FastTree/TreeTrainersCatalog.cs

```diff
@@ -437,5 +436,45 @@ public static class TreeExtensions
             var env = CatalogUtils.GetEnvironment(catalog);
             return new FastForestBinaryTrainer(env, options);
         }
+
+        public static PretrainedTreeFeaturizationEstimator PretrainTreeEnsembleFeaturizing(this TransformsCatalog catalog,
+            PretrainedTreeFeaturizationEstimator.Options options)
+        {
+            Contracts.CheckValue(catalog, nameof(catalog));
+            var env = CatalogUtils.GetEnvironment(catalog);
+            return new PretrainedTreeFeaturizationEstimator(env, options);
+        }
+
+        public static FastForestRegressionFeaturizationEstimator FastForestRegressionFeaturizing(this TransformsCatalog catalog,
```

Comment:
This naming pattern reads a little funny. How about turning it into `FeaturizeXXX`? Like we have with `FeaturizeText`.

Reply:
I can do `FeaturizeBy...`.

Reply:
That sounds better (to me at least).